modules/sbo-release-notes-1-2.adoc

```diff
@@ -0,0 +1,64 @@
+[id="sbo-release-notes-1-2_{context}"]
+// Module included in the following assembly:
+//
+// * applications/connecting_applications_to_services/sbo-release-notes.adoc
+:_content-type: REFERENCE
+= Release notes for {servicebinding-title} 1.2
+
+{servicebinding-title} 1.2 is now available on {product-title} 4.7, 4.8, 4.9, 4.10, and 4.11.
+
+[id="new-features-1-2_{context}"]
+== New features
+This section highlights what is new in {servicebinding-title} 1.2:
+
+* Enable {servicebinding-title} to consider optional fields in the annotations by setting the `optional` flag value to `true`.
+* Support for `servicebinding.io/v1beta1` resources.
+* Improvements to the discoverability of bindable services by exposing the relevant binding secret without requiring a workload to be present.
+
+[id="known-issues-1-2_{context}"]
+== Known issues
+* Currently, when you install {servicebinding-title} on {product-title} 4.11, the memory footprint of {servicebinding-title} increases beyond expected limits. But with low usage, the memory footprint stays within the expected ranges of your environment or scenarios. In comparison with {product-title} 4.10, under stress, both the average and maximum memory footprint increase considerably. This issue is evident in the previous versions of {servicebinding-title} as well. There is currently no workaround for this issue.
```

Comment:
@dperaza4dustbit and @pmacik: PTAL at this newly added Known Issue and provide feedback. 

Please provide the APPSVC JIRA link to this issue. I will add it here. Thanks!

Reply:
Hi @Srivaralakshmi the JIRA issue is [APPSVC-1200](https://issues.redhat.com/browse/APPSVC-1200).

Reply:
@pmacik Thanks for the issue link. Does the RN Text look good? If yes, then pls provide your LGTM in the PR and I will add it to the JIRA.

Reply:
Done.